finish/src/main/java/io/openliberty/guides/inventory/model/InventoryList.java

```diff
@@ -15,12 +15,15 @@
 import java.util.List;
 
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
-
```

Comment:
Keep this spacing between imports and beginning of class annotation

Reply:
Updated.